test/opal/test_openuri.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: false
+require "test/unit"
+require "open-uri"
+
+class TestOpenURI < Test::Unit::TestCase
+  def test_open_https
+    open("https://opalrb.com/") {|f|
+      assert_match(/Opal: Ruby to JavaScript Compiler/, f.read)
```

Comment:
So we can't change this text on the website? 😄

Reply:
lol! 😆 
We should probably start a local http server to not rely on network but I was sure how to do it

Reply:
@iliabylich The original test file is using `webrick`, do you think we should add this development dependency ?

Reply:
Yes. Otherwise the test becomes unreliable. For example, `opal-browser` uses [this](https://github.com/opal/opal-browser/blob/master/config.ru#L12) server to test HTTP module.

Reply:
Thanks for the pointer, we are now using Sinatra :tada: